manage_cursor.sh

```
@@ -1,138 +1,285 @@
-    # --- Download Icon and create Desktop Entry ---
-    #read -p "Enter the icon filename from GitHub (e.g., cursor-icon.png): " ICON_NAME_FROM_GITHUB
-    #ICON_DOWNLOAD_URL="https://raw.githubusercontent.com/hieutt192/Cursor-ubuntu/main/images/$ICON_NAME_FROM_GITHUB"
-    ICON_DOWNLOAD_URL="https://raw.githubusercontent.com/cursor/docs/refs/heads/main/images/logo/logo-transparent.png"
-    echo "Downloading icon to $ICON_PATH..."
-    sudo curl -L "$ICON_DOWNLOAD_URL" -o "$ICON_PATH"
+    figlet -f slant "Install Cursor"
+    echo "💿 Installing Cursor AI IDE on Ubuntu..."
+
+    install_dependencies
+
+    local cursor_download_path=$(get_appimage_path "install")
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)